docs/api.md

```diff
@@ -475,6 +476,99 @@ Content:
 
 Students are identified by the `client_id` Cookie.
 
+
+### <a name="get-engagement"></a>Get Engagement
+
+Get a lectures engagement values
+
+#### URL
+
+`/lectures/:lecture-id/engagements`
+
+#### Method
+
+GET
+
+#### URL Parameters
+
+##### Optional
+
+* `last=[integer]` 1 or 0. 1 shows only last entry for each student.
+* `recent=[integer]` 1 or 0. 1 shows only recent entries (last 15 min)
+
+#### Success Responses
+
+##### Success
+
+Code: 200
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "engagement": {
+            "description": "List of all engagements",
+            "type": "array",
+            "items": {
+                "description": "One engagement point",
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "description": "ID of engagement point",
+                        "type": "integer"
+                    },
+                    "userID": {
+                        "description": "ID of user",
+                        "type": "integer"
+                    },
+                    "interest": {
+                        "description": "The amount of interest in range [0, 1]",
+                        "type": "number"
+                    },
+                     "challenge": {
+                        "description": "The amount of challenge in range [0, 1]",
+                        "type": "number"
+                    },
+                    "submissionTime": {
+                        "description": "Time point when engagement point was submitted in ISO format (ISO 8601)"
+                        "type": "string"
+                    }
+                }
+            }
+        }
+    }
+}
+
+```
+
+#### Error Responses
+
+##### Resource Not Found
+
+The lecture was not found.
+See error message for details.
+
+Code: 404
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "message": {
+            "description": "Error message",
+            "type": "string"
+        }
+    }
+}
+```
+
+#### Notes
+
+last and recent are set by `/lectures/:lecture-id/engagements?last=0.25&recent=0.44`
```

Comment:
In the parameter specification you say 0 or 1, but here you have 0.25 and 0.44 which confuses me.